tests/conftest.py

```diff
@@ -173,6 +175,13 @@ def temp_testbed(monkeypatch):
     shutil.rmtree(temp_dir, onerror=add_permissions)
 
 
+# Always set the user config to just be a config in the temp_testbed; that way,
+# it will be unset unless a specific test wants to make a config in the testbed
+@pytest.fixture(autouse=True)
```

Comment:
I use the config to change the model for the exercism benchmark. Is there an easy way to turn off this fixture for just that test?

Reply:
This only mocks the user config set in ~/.mentat; so you can still change the project config and it should work just fine

Reply:
If you need to change the model in a test itself, you can check out the replacement_parser_test fixture I have that sets the model there to see how I change the config for specific tests (although setting a user config in the temp_testbed, now that the test user_config is automatically set there, might be a better way to go about it)